### ISSUE TITLE
how to give url pattern to search for values like "L76255 - Manual Call Point"  . values which contains whitespaces ?

### ISSUE THREAD

**Issue by rishi1804** [ziishaned/learn-regex#194]

**Comment by loisaidasam** [comment]
Hi @rishi1804 , can you provide more detail?

So the pattern can be letters, numbers, dashes, and spaces? Any specific order?

Maybe something like:

```
[\w\-\s]+
```

(Some amount of alphanumeric characters/hyphens/spaces)

🤷‍♀️

**Comment by rishi1804** [comment]
@loisaidasam  yes the pattern can consist of letters, numbers, dashes, and spaces, for example, I need to match values like  L76255 - Manual Call Point   while making URL patterns in Django

**Comment by rishi1804** [comment]
def get_asset_deatils_by_class(request,slug):
    connections.create_connection(hosts=['localhost'], timeout=20)
    client = Elasticsearch()
    s = Search(using=client, index='assets').query("match",class__raw=slug)
    response = s.execute()
    list1 = response.to_dict()
    return JsonResponse(list1)


this is the code and here the slug argument is going to be of the form L76255 - Manual Call Point
